code/modules/orbit/orbit.dm

```diff
@@ -33,7 +33,7 @@
 			if (!orbiting.orbiters.len)//we are the last orbit, delete the list
 				orbiting.orbiters = null
 		orbiting = null
-	..()
+	return ..()

```

Comment:
I think it was causing `QDEL_HINT_QUEUE` to not get passed all the way to when `qdel(orbiting)` is called in `stop_orbit()`